src/plguins/useCellRangeSelection.js

```diff
@@ -211,10 +211,26 @@ function useInstance (instance) {
     [allColumns, cellsById, cellIdSplitBy, rows]
   )
 
```

Comment:
Tried to add `selectedFlatCells` with `cell.isSelected.`

`selectedFlatCells ` had `cell.isSelected` in it, but while we loop `row.cells` on UI file (app.js), it was missing